interface/src/app/gene/network/network.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * "adage.gene.network" module.
+ */
+
+angular.module('adage.gene.network', [
+  'ui.router',
+  'placeholders',
+  'ui.bootstrap',
+  'ui.slider'
+])
+
+.config(function config($stateProvider) {
+  $stateProvider.state('gene_network', {
+    url: '/gene_network/?genes&mlmodel',
+    views: {
+      main: {
+        templateUrl: 'gene/network/network.tpl.html',
+        controller: 'GeneNetworkCtrl',
+        controllerAs: 'ctrl'
+      }
+    },
+    data: {pageTitle: 'Gene Network'}
+  });
+})
+
+.factory('EdgeService', ['$resource', function($resource) {
+  return $resource('/api/v0/edge/');
+}])
+
+.factory('NodeService', ['$resource', function($resource) {
+  return $resource('/api/v0/node/');
+}])
+
+.controller('GeneNetworkCtrl',
+   ['$stateParams', 'EdgeService', 'NodeService', '$log',
+    function GeneNetworkController($stateParams, EdgeService, NodeService,
+                                   $log) {
+      var rawMinWeight = -1.0;
+      var rawMaxWeight = 1.0;
+      var scaledMaxWeight = rawMaxWeight - rawMinWeight;
+      var self = this;
+      // The following properties of "self" will be available to HTMl.
```

Comment:
Totally minor, nitpicky typo: is that last "l" in HTML supposed to be capitalized?